libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -286,17 +272,23 @@ void AP_Logger_File::Prep_MinSpace()
         return;
     }
 
+    const int64_t total = disk_space();
+    if (total == -1) {
+        return;
+    }
+    const int64_t target_free = MIN(_front._params.min_MB_free * 1000,total*MAX_FREE_RATIO);
```

Comment:
Whitespace issue

Reply:
This is a significant behavioural change, deleting more files than to get us to the min-free amount.  Do we need it?

Reply:
This should also be unsigned, surely?  Casting may be required.

Reply:
Fixed white space, I think. By default its only deleting more on a 4GB card. You would remove MAX_FREE_RATIO? I wasn't sure what would happen if we tried to delete everything / free more space than we had. Removing it would make it quite a bit simpler.